### ISSUE TITLE
demographic / eda corrections

### ISSUE THREAD

**Issue by EmilyPo** [EmilyPo/Diss-EPT#3]
Correction Types / Notes

thinking through necessary corrections for my stergms following:
demography writeup (https://github.com/statnet/CrossProjectModeling/issues/9)
eda writeup (https://github.com/statnet/CrossProjectModeling/issues/10)

*1. Issue w/ changing age & not forming sufficient nodes*
- could try to use function-based approach but...
- my formation models are somewhat complex w/ age and sex differences so manual calibration of specific formation coefs probably best way to go 

*2. "true" duration estimation too account for ties lost due to deaths/aging-out*
- implies that estimated "exit rate" should not be used in addition to duration in fit -> netest conversion using EDA (correct?) It's kind of the same idea but using a simulation-based method instead, and avoid current death rate issue where we need corrected exit rates to account for different networks and likelihood that edges are broken due to exits 
- will definitely need to do this for both marriage/cohab and causal networks but will have more effect for marriage/cohab

*3. EDA aka log(d) vs log(d-1)*
- yes should implement esp for casual network. mean degree in marriage/cohab network is actually close to 0.5 so may not make a huge difference but should implement for consistency's sake
- will need to adjust ego.netest function

**Comment by EmilyPo** [comment]
@chad-klumb 
could I take a look at the code you used in the demography writeup as a guide? I'd like to try and implement the in-simulation persistence correction for my own networks

**Comment by chad-klumb** [comment]
Sure, can I get it to you later this week?  It will probably benefit from some cleanup and explanation (since it wasn't written with external audiences in mind).

**Comment by EmilyPo** [comment]
Of course! Next week is fine too - I’ve got plenty of work to keep me busy. Thanks.

**Comment by martinamorris** [comment]
fyi, @smjenness is working on implementing the EDApprox correction in EpiModel now.

**Comment by smjenness** [comment]
> fyi, @smjenness is working on implementing the EDApprox correction in EpiModel now.

I have assigned to @chad-klumb
